### ISSUE TITLE
Laggy drag with data-wails-drag

### ISSUE THREAD

**Issue by brys0** [wailsapp/wails#1731]
### Description

The drag mechanic used when providing your own drag mechanic *lags behind* or is delayed of dragging. 
For example holding down the mouse and dragging quickly enough causes the mouse to freeze on a non-drag reason. But the app to still be dragged from that region

### To Reproduce

Create a drag region of your choice. Quickly hold your mouse down and drag. It should either stop moving outside the window or where it shouldn't be able to drag. 

### Expected behaviour

When holding the mouse down on the given drag region it should freeze on that position of the window with no delay. 

### Screenshots

This video effectively displays the issue with dragging the window
https://user-images.githubusercontent.com/35676267/184232472-c1bef5ae-cff9-416e-8489-e344148c19e5.mp4

### Attempted Fixes

Disabling tabbed window effect does not work
Setting solid background color does not work
Disabling transparent window does not work

Setting window to `Frameless` -> `false` and using inbuilt drag region does work

### System Details

```shell
System
------
OS:             Windows 10 Home
Version:        2009 (Build: 22621)
ID:             22H2
Go Version:     go1.18.5
Platform:       windows
Architecture:   amd64

Wails
------
Version:        v2.0.0-beta.43

Dependency      Package Name    Status          Version
----------      ------------    ------          -------
WebView2        N/A             Installed       104.0.1293.47
npm             N/A             Installed       8.5.2
*upx            N/A             Available
*nsis           N/A             Installed       v3.08

* - Optional Dependency

Diagnosis
---------
Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
```

**Comment by leaanthony** [comment]
This chestnut 😬 Seems like we bounce between double click events and smooth drag. The crux of the problem is that webview2 doesn't forward events to the host application so we have to process events in JS, which isn't ideal. Microsoft are currently testing a fix for this but it's not released yet.

The last Wails release had some improvements in this area (now there's no delay before dragging) but looks like this is an edge case.

 I'm thinking, as a workaround, we could disable the double click support by default and have an optimal solution for that. If you enable doubleclick support then you get the current solution when "fast drag" is not optimal.

Thoughts?

**Comment by brys0** [comment]
Seems like if you don't wait before dragging it just doesn't stop the cursor until 50-100ms 😂. Disabling doubleclick by default would def be nice.. I'm not sure who uses it. (I'm pretty sure no one) but that would be lovely.

**Comment by brys0** [comment]
#1732 Is my quickly thrown together pr that should solve the lag with dragging at least.

**Comment by leaanthony** [comment]
Please check https://github.com/wailsapp/wails/pull/1704 for reference on previous work on this 👍

**Comment by leaanthony** [comment]
Can't reproduce
https://user-images.githubusercontent.com/1943904/184609883-29f5943f-e8cf-4907-97aa-3aac3dec10bf.mp4

**Comment by brys0** [comment]
Try it with a frameless window.

**Comment by leaanthony** [comment]
Same

**Comment by brys0** [comment]
Maybe try it with transparent & tabbed window? (It could be performance issues...) Otherwise it might be my version of windows or WebView I'm not sure

**Comment by brys0** [comment]
Also, with this the way I implement the drag region might also effect this. I make it a fixed display that covers around 3/4th of the screen it's under about 5 other elements. It sounds weird but this **could** affect performance in doing it's border calculations. Just a thought 💭

**Comment by leaanthony** [comment]
@brys0 Please try this PR: https://github.com/wailsapp/wails/pull/1750. 
There's a [guide](https://wails.io/docs/guides/bleeding-edge#testing-a-pr) for testing PRs. Let us know how you get on 👍

**Comment by leaanthony** [comment]
Merged #1750 - will become default in release +1

**Comment by leaanthony** [comment]
@brys0 - The PR here makes the property and value customisable. This should get around the dynamic styling issue you mentioned: #1828
